docker/requirements.in

```diff
@@ -24,7 +24,7 @@ click
 dask[complete]
 dask_image
 
-datacube[dev]<=1.8.9
```

Comment:
Honestly, this freaks me out a bit.  Why was it pinned <=1.8.9?  Why was it pinned <= anything and what happened at 1.8.10? 

....  Oh that's right, 1.8.10 was broken.  Should really have been "!=1.8.10"